api-metastore/src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -558,6 +561,18 @@ public EventType get(final String eventTypeName) throws NoSuchEventTypeException
         }
     }
 
+    private EventTypeStatistics validateStatisticsUpdate(
```

Comment:
I don't see why this method should return something. 
It could simply check `if (newStatistics is != null && !Objects.equals(existing, newStatistics))` 
and throw an exception in this case. 
And the calling method above can always use `existing`, when the validation is successful.

Reply:
Makes total sense a9014f384